qfp_wizard.py

```diff
@@ -127,32 +134,33 @@ def BuildThisFootprint(self):
         bottom_edge = (v_pitch + pad_length) / 2
         top_edge = -bottom_edge
 
+        self.draw.SetLineThickness( pcbnew.FromMM( 0.1 ) ) #Default per KLC F5.2 as of 12/2018
         self.draw.BoxWithDiagonalAtCorner(0, 0, w, h, bevel)
 
         # Draw silkscreen
-        self.draw.SetLayer(pcbnew.F_SilkS)
+        self.draw.SetLayer( pcbnew.F_SilkS )
+        self.draw.SetLineThickness( pcbnew.FromMM( 0.12 ) ) #Default per KLC F5.1 as of 12/2018
 
         #top left - as per IPC-7351C
-        self.draw.Polyline([(-inner, -y), (-x, -y), (-x, -inner), (left_edge, -inner)])
+        self.draw.Polyline([(-innerX, -y), (-x, -y), (-x, -innerY), (left_edge, -innerY)])
         # top right
-        self.draw.Polyline([(inner, -y), (x, -y), (x, -inner)])
+        self.draw.Polyline([(innerX, -y), (x, -y), (x, -innerY)])
         # bottom left
-        self.draw.Polyline([(-inner, y), (-x, y), (-x, inner)])
+        self.draw.Polyline([(-innerX, y), (-x, y), (-x, innerY)])
         # bottom right
-        self.draw.Polyline([(inner, y), (x, y), (x, inner)])
+        self.draw.Polyline([(innerX, y), (x, y), (x, innerY)])
 
         # Courtyard
         cmargin = self.parameters["Package"]["courtyard margin"]
-        self.draw.SetLayer(pcbnew.F_CrtYd)
-        sizex = (right_edge + cmargin) * 2
-        sizey = (bottom_edge + cmargin) * 2
+        self.draw.SetLayer( pcbnew.F_CrtYd )
+        sizex = ( right_edge + cmargin ) * 2
```

Comment:
Don't add extra spaces here unless you are going to be consistent with them through the file.